vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/validation/ComboBoxBinderValidationIT.java

```diff
@@ -0,0 +1,89 @@
+package com.vaadin.flow.component.combobox.test.validation;
+
+import com.vaadin.flow.component.combobox.testbench.ComboBoxElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.validation.AbstractValidationIT;
+import org.junit.Test;
+import org.openqa.selenium.Keys;
+
+import static com.vaadin.flow.component.combobox.test.validation.ComboBoxBinderValidationPage.ENABLE_CUSTOM_VALUE_BUTTON;
+import static com.vaadin.flow.component.combobox.test.validation.ComboBoxBinderValidationPage.EXPECTED_VALUE_INPUT;
+import static com.vaadin.flow.component.combobox.test.validation.ComboBoxBinderValidationPage.REQUIRED_ERROR_MESSAGE;
+import static com.vaadin.flow.component.combobox.test.validation.ComboBoxBinderValidationPage.UNEXPECTED_VALUE_ERROR_MESSAGE;
+
+@TestPath("vaadin-combo-box/validation/binder")
+public class ComboBoxBinderValidationIT
+        extends AbstractValidationIT<ComboBoxElement> {
+
+    @Test
+    public void fieldIsInitiallyValid() {
+        assertClientValid();
+        assertServerValid();
+        assertErrorMessage(null);
+    }
+
+    @Test
+    public void required_triggerInputBlur_assertValidity() {
+        testField.sendKeys(Keys.TAB);
+        assertServerInvalid();
+        assertClientInvalid();
+        assertErrorMessage(REQUIRED_ERROR_MESSAGE);
+    }
+
+    @Test
+    public void required_changeValue_assertValidity() {
+        $("input").id(EXPECTED_VALUE_INPUT).sendKeys("foo", Keys.ENTER);
```

Comment:
I added custom validation to the Binder tests, as when allowing custom values, the user is mostly free to enter anything. This test does not allow custom values, just testing it here as well for completeness sake. 

Also the test process differs a bit from the input field ones: For the input fields, custom validation was tested as part of the constraint tests. Combo box does not have any constraints, so I attempted to add this to the required tests.